pkgs/tools/compression/ouch/default.nix

```diff
@@ -12,36 +12,34 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "ouch";
-  version = "0.3.1";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = "ouch-org";
     repo = pname;
     rev = version;
-    sha256 = "sha256-I9CgkYxcK+Ih9UlcYBa8QAZZsPvzPUK5ZUYKPxzgs38=";
+    sha256 = "sha256-XB0J7Qeru+FX5YprepglfTndS8b3zsAw1b9mc4n6EdA=";
   };
 
-  cargoSha256 = "sha256-jEprWtIl5LihD9fOMYHGGlk0+h4woUlwUWNfSkd2t10=";
+  cargoSha256 = "sha256-aW1aDXxs64ScocrnlsGy2+NAs6aC8F0/S1f32f9BDJU=";
 
-  nativeBuildInputs = [ help2man installShellFiles pkg-config ];
+  nativeBuildInputs = [ installShellFiles pkg-config ];
 
   buildInputs = [ bzip2 xz zlib zstd ];
 
   buildFeatures = [ "zstd/pkg-config" ];
 
   postInstall = ''
-    help2man $out/bin/ouch > ouch.1
-    installManPage ouch.1
-
-    completions=($releaseDir/build/ouch-*/out/completions)
-    installShellCompletion $completions/ouch.{bash,fish} --zsh $completions/_ouch
+    installManPage artifacts/*.1
+    installShellCompletion artifacts/ouch.{bash,fish} --zsh artifacts/_ouch
   '';
 
-  GEN_COMPLETIONS = 1;
+  OUCH_ARTIFACTS_FOLDER = "artifacts";
```

Comment:
What does setting it up do ? Probably adding a comment would be helpful.

Reply:
it makes the build script build completions and man pages